Add public_network_access_enabled property to app service docs (#22399)
hashicorp/terraform-provider-azurerm: website/docs/r/linux_function_app.html.markdown

* `https_only` - (Optional) Can the Function App only be accessed via HTTPS? Defaults to `false`.

> * `public_network_access_enabled` - (Optional) Whether or not public network access is enabled for the Function App. Defaults to `true`.

Reviewer: ```suggestion
* `public_network_access_enabled` - (Optional) Should public network access be enabled for the Function App. Defaults to `true`.
```
